test/ibmq/test_registration.py

```diff
@@ -102,22 +103,45 @@ def test_environ_over_qiskitrc(self):
 class TestCredentialsKwargs(IBMQTestCase):
     """Test for ``Credentials.connection_parameters()``."""
 
-    def test_no_proxy_params(self):
+    def test_no_proxy_params(self) -> None:
         """Test when no proxy parameters are passed."""
         no_params_expected_result = {'verify': True}
         no_params_credentials = Credentials('dummy_token', 'https://dummy_url')
         result = no_params_credentials.connection_parameters()
         self.assertDictEqual(no_params_expected_result, result)
 
-    def test_verify_param(self):
+    def test_verify_param(self) -> None:
         """Test 'verify' arg is acknowledged."""
         false_verify_expected_result = {'verify': False}
         false_verify_credentials = Credentials(
             'dummy_token', 'https://dummy_url', verify=False)
         result = false_verify_credentials.connection_parameters()
         self.assertDictEqual(false_verify_expected_result, result)
 
-    def test_proxy_param(self):
+    def test_options_param(self) -> None:
+        """Test options param is acknowledged."""
+        # Iterate through qiskitrc credentials
+        for auth, i in enumerate(read_credentials_from_qiskitrc()):
+            if not isinstance(auth, OrderedDict):
+                continue
```

Comment:
I don't think a real `qiskitrc` even exists in the CI environment. A better test would be to use a custom qiskitrc (see `custom_qiskitrc()` in this module), store test data and retrieve it. In addition, there should be tests on `provider.experiment.set_option()` etc.

Reply:
The tests were removed from `test_registration` to `test_experiment`. They use the same ops, so the coverage should be identical, plus now it includes the `ExperimentService` functions.